clicommand/agent_start.go

```diff
@@ -297,6 +299,11 @@ var AgentStartCommand = cli.Command{
 			Usage:  "Prepend timestamps on each line of output.",
 			EnvVar: "BUILDKITE_TIMESTAMP_LINES",
 		},
+		cli.StringFlag{
+			Name:   "health-check-addr",
+			Usage:  "Start an HTTP server on this addr:port that returns whether the agent is healthy, disabled by default",
+			EnvVar: "BUILDKITE_HEALTH_CHECK_ADDR",
```

Comment:
For modern params we use `BUILDKITE_AGENT_` as a prefix, it's long, but let's go with `BUILDKITE_AGENT_HEALTH_CHECK_ADDR`

Reply:
Turns out, I'm totally wrong :D